java/fury-core/src/main/java/org/apache/fury/AbstractThreadSafeFury.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.fury;
```

Comment:
Please properly add license header or include necessary info if it's copied.

Reply:
header added.  Thanks for this catch, I always forget add header for new file.

Reply:
I don't look into details but CI should fail if license headers missing.

Reply:
* the code style check was failing until the header was added
* the Graal VM CI is failing still

Reply:
CI succeed now